gulp-tasks/utils/external-helpers.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
+// This is extracted from the Babel runtime, meant to be included once in
+// workbox-core and reused throughout all of the other modules as a way of
+// avoiding code duplication.
+// See https://github.com/babel/babel/blob/9e0f5235b1ca5167c368a576ad7c5af62d20b0e3/packages/babel-helpers/src/helpers.js#L240
```

Comment:
This link doesn't explain much.

What is this code doing, why is it in the  repo and why isn't babel preset env not adding this for us?

Reply:
From the comments at least, it isn't clear why we're extracting this from the Babel runtime. If this is for a workaround because babel-preset-env is unable to inject for us, it would be good to clarify that in the comments.